Extensions/Lighting/lightruntimeobject.js

```diff
@@ -90,14 +81,8 @@ gdjs.LightRuntimeObject.prototype.updateFromObjectData = function (
   }
 
   if (oldObjectData.content.texture !== newObjectData.content.texture) {
-    this._texture =
-      newObjectData.content.texture === ''
-        ? null
-        : this._runtimeScene
-            .getGame()
-            .getImageManager()
-            .getPIXITexture(newObjectData.content.texture);
-    this._renderer.updateTexture();
+    this._texture = newObjectData.content.texture;
+    this._renderer.updateMesh();
```

Comment:
You're not calling updateTexture?

Reply:
I'll have to update the entire mesh to update texture, since I'll have to swap the shader in this case.

Reply:
Sounds good then!